onnxruntime/python/tools/transformers/onnx_model.py

```diff
@@ -906,6 +908,31 @@ def remove_unused_constant(self):
         if len(unused_nodes) > 0:
             logger.debug(f"Removed unused constant nodes: {len(unused_nodes)}")
 
+    def get_subgraph_inputs_of_node(self, node):
```

Comment:
Can we change this to private function (add _ prefix)? I think it is not ready as public API.
For example, attr.graphs will also contain subgraphs.